### DISCUSSION TITLE
Way to earn Star

### DISCUSSION THREAD

**Discussion by emptyfist** [community/community#30917] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello everyone.

Always appreciate your help.
I'm very proud of other git profile which earns `Star`.

Plz, let me know how can I get `Star` and what is necessary or usage of `List`?

**Comment by melvincwng** [comment, marked as answer]
Hi @shinagawa-haruko , welcome to GitHub Community  👋🏻 こんにちは!

Regarding `Stars`, you can gain them in your code repositories by creating awesome projects & sharing them with your family/friends/online. It's even better if those code repositories are of actual utility/usage to developers. From what I've noticed so far, repositories with lots of stars tend to provide some use case that solves a particular problem in the world. 

Take for example - [Bootstrap](https://github.com/twbs/bootstrap). This open source code repository is not only well known amongst the developer community, it actually provides great utility usage as well! Need a slick carousel or button?  Just import it from Bootstrap! So convenient, and hence that's why they have **159k stars.** Because they help to solve a real world problem.

Regarding `Lists`, they are essentially a feature provided by GitHub that allows you to collate and organize all your favourite repositories in one place! If you need more information, please refer to this [GitHub Documentation](https://docs.github.com/en/get-started/exploring-projects-on-github/saving-repositories-with-stars#organizing-starred-repositories-with-lists).

Hope this answers your question (これがあなたの質問に答えることを願っています)

Take care, and have a good day ahead 😄

**Reply by emptyfist** [reply]
こんにちは。
ありがとうございました。
多くの助けてくれありがとう。
